packages/frontend/src/app/home/home.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {HomeComponent} from './home.component';
+import {Component} from '@angular/core';
+
+// Mock the inner components
+@Component({
+  selector: 'app-search',
+})
+class SearchComponent {}
+
+describe('HomeComponent', () => {
+  let component: HomeComponent;
+  let fixture: ComponentFixture<HomeComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [HomeComponent, SearchComponent],
+      imports: [MatToolbarModule],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(HomeComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create the component', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it("should have as title 'flaky-dashboard'", () => {
+    expect(component.title).toEqual('flaky-dashboard');
+  });
+
+  it('should render title', () => {
+    fixture.detectChanges();
+    const compiled = fixture.nativeElement;
+    expect(compiled.querySelector('span').textContent).toContain(
+      'flaky-dashboard is running !'
```

Comment:
Assertions like this will over time need to keep being updated, not a blocker for this PR, but it might be worth looking at [snapshot testing](https://izifortune.com/snapshot-testing-angular-applications/) eventually.